assets/app/view/game/map_controls.rb

```diff
@@ -55,6 +58,69 @@ def starting_map_controls
         render_button(text, on_click)
       end
 
+      def generate_last_route(entity)
+        trains = entity.trains
+        operating = entity.operating_history
+        last_run = operating[operating.keys.max]&.routes
+        return [] unless last_run
+
+        halts = operating[operating.keys.max]&.halts
+        routes = []
+        last_run.map do |train, connections|
+          next unless trains.include?(train)
```

Comment:
Do you really want this? Doesn't this mean that a route won't be shown if the corporation no longer owns the train with which the route was run?